src/App.tsx

```diff
@@ -24,9 +24,7 @@ function App() {
         height: backlightContainer.clientHeight,
       });
     window.addEventListener('resize', handleResize);
-    handleResize();
-
-    return () => window.removeEventListener('resize', handleResize);
+    handleResize();    return () => window.removeEventListener('resize', handleResize);
```

Comment:
what happened to this new line?

Reply:
99% chance i was toying with vim too fast...